### PR TITLE
multiple contests

### LINKED ISSUE #106: Multiple contests
Right now our system provides the ability to host a single contest at a time. We need to include the ability to host multiple contests at the same time.

For example, one contest can be a simple practice contest so that people can see how the system works. Then the real contest can start based on the time setting implementation for #103

### LINKED ISSUE #107: Multiple contest UI design
The UI will need to account for the fact that multiple contests can be hosted at the same time.

Thus #104 will need to change to show multiple contests somehow.

### PR DESCRIPTION
**Pull request by rishabhKalakoti**
- made a basic system for multiple contests
- made a basic UI for multiple contests
- dummy contests to be added manually as of now
- contest timings not covered in this PR
- exception handler missing
- same question set copied again to have more questions for testing

fixes #106 
fixes #107

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -26,10 +43,26 @@
...
+    return template("dashboard.html", contests=contests)
+
+@app.get("/contest/<code>/<number>")
+def contest(code,number):
+    statement = questions[number].statement
+    return template("index.html", question_number=number, question=statement)
+
+@app.get("/contest/<code>")
+def contest(code):
+    for contest in contests:
```
Comment (theSage21) [inline server.py]: wouldn't it be nice if the contests list was a dictionary indexed by code instead? :thinking:

**Comment by theSage21** [comment]
This is beautiful work. :+1:

Some stuff needs to be filled out. Would you like to do it as separate issues or within this PR? Here are things that come to mind:

- [x] reusing questions per contest is actually a good idea
- [ ] leaderboard calculation needs to change for each contest
- [ ] leaderboard calculation needs to only count questions which are in the contest

**Comment by rishabhKalakoti** [comment]
I was planning to go like:
multiple contests > contest timings > leaderboard
one by one to keep things simpler

File: server.py (hunk 1 above)
Comment (rishabhKalakoti) [reply server.py]: I was thinking the same, just did not implement it earlier for simplicity and forgot. Will do it in a while.

**Pull request merged**